LICENSE.txt

```diff
@@ -0,0 +1,748 @@
+A. HISTORY OF THE SOFTWARE

```

Comment:
I think you did not mean to add this license text here at the root, correct?

Reply:
done